fate_flow/manager/tracking.py

```diff
@@ -156,7 +156,7 @@ def get_metric_list(self, job_level: bool = False):
                         'and f_task_id = "{}"'.format(
                 self.get_table_index(), self.job_id, self.component_name if not job_level else 'dag', self.role,
                 self.party_id, self.task_id)
-            stat_logger.info(query_sql)
+            # stat_logger.info(query_sql)
```

Comment:
Please delete or change to debug, the following lines is also

Reply:
done